src/services/weather.js

```diff
@@ -30,6 +34,28 @@ async function fetchWeatherData(city) {
   return mapModelToView(response.data);
 }
 
-function getIconURL(icon) {
+export function getIconURL(icon) {
   return weatherImageURL + icon + imageExtension;
 }
+
+function mapModelForecastToView(model) {
+  return _.map(model, mapModelToView);
+}
+
+export const fetchWeatherForcastForCity = async (city, completation) => {
```

Comment:
convert to regular function. It can be an arrow function as well, but all the others in this file are regular functions so this one should be too :)